scripts/west_commands/sign.py

```diff
@@ -420,19 +420,34 @@ def sign(self, command, build_dir, bcfg, formats):
         board = cache['CACHED_BOARD']
         log.inf('Signing for board ' + board)
         target = self.edt_get_rimage_target(board)
-        log.inf('Signing for SOC target ' + target)
+        conf = target + '.toml'
+        log.inf('Signing for SOC target ' + target + ' using ' + conf)
 
         if not args.quiet:
             log.inf('Signing with tool {}'.format(tool_path))
 
+        s = pathlib.Path(os.environ.get('ZEPHYR_BASE'))
+
         bootloader = str(b / 'zephyr' / 'bootloader.elf.mod')
         kernel = str(b / 'zephyr' / 'zephyr.elf.mod')
         out_bin = str(b / 'zephyr' / 'zephyr.ri')
+        out_xman = str(b / 'zephyr' / 'zephyr.ri.xman')
+        out_tmp = str(b / 'zephyr' / 'zephyr.rix')
+        conf_path = str(s / '..' / 'modules' / 'audio' / 'sof' / 'rimage' / 'config' / conf)
```

Comment:
Hard-coding the module location like this isn't so nice. How about something like storing its location in the cmake cache instead?

Reply:
@mbolivar-nordic thanks! Is this version better? It requires https://github.com/thesofproject/sof/pull/3733 to go with it

Reply:
https://github.com/thesofproject/sof/pull/3733 has been merged